book/version_control.md

```diff
@@ -33,14 +34,14 @@ When used effectively, version control helps us to identify which changes have n
 
 Git is invaluable when recording and combining changes from multiple developers, as merging allows us to resolve conflicts between overlapping changes. Using a remote Git repository maintains a single source of truth, even when multiple individuals are working on a project. Additionally, version control platforms, like GitHub and GitLab, can make it easier to track and review ongoing work. This avoids duplication of effort and keeps review embedded in the development workflow.
 
-
-## What should I version control?
+## Include or exclude
```

Comment:
We should make this more directed (e.g. exclude this). If the section covers including and excluding we should split it into two

Reply:
Added new issue as I think we should split this as you suggest.